### ISSUE TITLE
"Current Containers" should give more info

### ISSUE THREAD

**Issue by sshipway** [itzg/minecraft-container-yard#3]
If possible, the Current list should also list the world seed or save filename if used, and the Difficulty/Mode.

I suggest these (plus the MOTD, difficulty, mode etc) are stored as labels against the container so as to be able to easily retrieve them (and to identify which containers are 'owned' by this manager)

**Comment by itzg** [comment]
Just wanted to note that `TYPE` also needs to be shown since it's not apparent this is a Forge server

![image](https://cloud.githubusercontent.com/assets/988985/12073079/c5d6e5cc-b0cd-11e5-8938-325760327fe3.png)

**Comment by sshipway** [comment]
Good point, and Version would be good as well.  In fact, if a Forge server, there should be a list of modules displayed (as clients may also need to have them installed) - possibly as a popup to conserve screen space?

**Comment by itzg** [comment]
Forge containers at least present the "mod pack" download option, but I do like the idea of listing them out to help describe the server.

**Comment by itzg** [comment]
Commit 9cdb018928406d98f268593c42e27be800f33834 added server type and version

**Comment by danpolanco** [comment]
Left to do, we have:
- [ ] difficulty / mode
- [ ] motd
- [ ] maybe a popup of modules / plugins

Anything else?

**Comment by sshipway** [comment]
World seed number (if it was specified, or if it was pulled from the World ZIP file).  
World name (from ZIP file if uploaded or taken from library)
Possibly world filename, though the name from the level.dat in the Zip file is probably sufficient

Agree that modules should be a popup.  
Maybe have whitelist and mods as popups if they are defined as well?

**Comment by sshipway** [comment]
Should also indicate if the 'public' flag is set.  

Possibly use a different coloured header to the section depending on public/private, or on Minecraft server type vanilla/bukkit/forge?

**Comment by itzg** [comment]
Yes, I was thinking that too. I might throw in a simple filtering of the container list to quickly see just public ones while logged in.
